pydata_sphinx_theme/__init__.py

```diff
@@ -154,9 +154,19 @@ def get_edit_url():
         if context.get("github_url"):
             github_url = context["github_url"]
 
-        github_user = context["github_user"]
-        github_repo = context["github_repo"]
-        github_version = context["github_version"]
+        github_user = context.get("github_user")
+        github_repo = context.get("github_repo")
+        github_version = context.get("github_version")
```

Comment:
Won't you need to update L144 otherwise it will error without the github settings?